pkg/api/qconfig.go

```diff
@@ -184,12 +184,9 @@ func NewDefaultAarch64Config(name string, numCpus uint32, numMemMB uint32, sockD
 				Path:    filepath.Join(sockDir, "monitor.sock"),
 			},
 		},
-		SerialDevices: []qcli.SerialDevice{
-			qcli.SerialDevice{
-				Driver:     qcli.PCISerialDevice,
-				ID:         "pciser0",
-				ChardevIDs: []string{"serial0"},
-				MaxPorts:   1,
+		LegacySerialDevices: []qcli.LegacySerialDevice{
+			qcli.LegacySerialDevice{
+				ChardevID: "serial0",
```

Comment:
I think we're going to want both; in case someone boots an image using console=ttyS0, and a different image with console=ttyAMA0.

I *think* we can fix this in qcli though; that is, if we use the PCISerialDevice and set Legacy, we should get:

```
-serial stdio \
-chardev socket,id=serial1,path=/tmp/console.sock,server=on,wait=off 
-device pci-serial,id=pciser0,chardev=serial1
```

Then we can have both -- this does mean that `machine console` will need some enhancement so one can select the different serial devices attached.  But that's OK.

Reply:
This can go the PR with the console path fix.